# Manual for the Chloroplast Modular Cloning System

## Type IIs restriction enzymes
<p>
Unlike conventional Type II restriction enzymes, which cleave DNA within their recognition sequences, Type IIS restriction enzymes cut outside their recognition sequences at defined distances (Figure 1). This property allows for the flexible design of overhangs, enabling a single enzyme to generate up to 256 distinct 4-base pair overhangs. This capability has led to the development of the Golden Gate cloning technique, now one of the most widely used cloning methods. Commonly used Type IIS enzymes include BsaI, BsmBI, BpiI (BbsI), PaqCI (AarI), and SapI.
</p>
<br>


<html>
<figure>
    <img src="../images/parts/Fig1.png"
         alt="General principle of Type IIs restriction enzymes">
    <figcaption>
<b>Figure 1</b> General principle of Type IIs restriction enzymes<br>
While Type II restriction enzymes cleave DNA within their recognition sequence, Type IIS restriction enzymes cut DNA adjacent to their recognition sites, allowing the 4-base pair cut site to be freely chosen. </figcaption>
</figure>
</html>
<br><br>


<p>
Golden Gate cloning allows these enzymes to be combined in a single "one-pot" reaction with T4 ligase, eliminating the need for separate restriction and ligation steps. Because the recognition sites for Type IIS enzymes are cleaved off during digestion and are not included in the final assembly (Figure 1), correctly assembled constructs are not cleaved further. This feature enables the cycling of restriction enzyme digestion and ligation steps, which increases the efficiency of obtaining correctly assembled constructs.
</p>
<br>

### General introduction
<p>
Following the initial development of Golden Gate cloning and the subsequent creation of the widely adopted Modular Cloning (MoClo) standard syntax, which employs standardized overhangs generated by Type IIS restriction enzymes such as BsaI, BpiI, and BsmBI, a consensus has emerged within the plant synthetic biology community. 
The widely adopted Phytobrick standard in the plant synthetic biology community has been successfully adapted for a range of photosynthetic organisms, including cyanobacteria, plants, mosses, plant chloroplasts, and even for nuclear engineering in Chlamydomonas.
To apply synthetic biology techniques to chloroplast engineering in Chlamydomonas reinhardtii, standardizing and characterizing genetic parts is essential. Although different cloning systems employ varying strategies for assembling multigene constructs, most are compatible with the assembly of single transcription units from standardized parts. Our system, utilizes a hierarchical assembly approach, similar to the common Plant MoClo and the Chlamydomonas MoClo system, rather than the iterative cloning scheme used in the GoldenBraid/Loop assembly standard.
</p>
<p>
The following sections provide a detailed explanation of our cloning system.
</p>
<br>

### The general Modular cloning scheme
<p>
In our hierarchical cloning scheme, individual Level 0 parts are first cloned into a universal acceptor vector. A Level 0 part represents the DNA sequence of a fundamental genetic element—such as a promoter, 5’UTR, coding sequence, or 3’UTR—flanked by a standardized 4-bp overhang and BsaI recognition sites.
</p>
<br>

<html>
<figure>
    <img src="../images/parts/Fig2.png"
         alt="The hierarchical modular cloning scheme">
    <figcaption>
<b>Figure 2</b> The hierarchical modular cloning scheme<br>
New parts are first domesticated and assembled into a universal acceptor vector (Level 0). These parts can then be combined in a one-pot reaction to create Level 1 plasmid containing a single transcriptional unit. Optionally, these single transcriptional units can be further assembled into a multigene cassette (Level 2).</figcaption>
</figure>
</html>
<br><br>

<p>
These parts are then assembled into a single transcriptional unit using a one-pot Golden Gate reaction with BsaI, combining 5–9 Level 0 parts into a Level 1 assembly. 
</p>

<p>
Next, 2–7 transcriptional units are integrated into a Level 2 multigene cassette via another one-pot Golden Gate reaction using BsmBI. In our system, the arrangement and order of transcription units are defined by the chosen connectors (detailed explanation below), rather than by the acceptor vectors used in the Level 1 assembly.
</p>

<p>
To enable selection of successfully assembled plasmids, different antibiotic markers are used at each assembly stage. In our system, Level 0 employs chloramphenicol, Level 1 uses carbenicillin (or ampicillin), and Level 2 utilizes kanamycin selection.
</p>
<br>

### Domesticating new Level 0 parts
<p>
New Level 0 parts are generated using PCR products or synthesized DNA. These linear sequences are then assembled into a single standardized universal acceptor vector. 
In contrast to other methods that require multiple acceptor vectors, only one is needed because standardized overhangs are incorporated into the PCR primers or synthesized sequence. The specific overhangs are detailed in Table 1.
</p>
<br>

<html>
<figure>
    <img src="../images/parts/Fig3.png"
         alt="Domesticating new Level 0 Parts ">
    <figcaption>
<b>Figure 3</b> Domesticating new Level 0 Parts <br>
PCR or DNA synthesis fragments that include standardized overhangs and BsmBI recognition sites are assembled into the Universal Acceptor vector. This vector features an mScarlet placeholder, enabling pink/white colony screening.</figcaption>
</figure>
</html>
<br><br>

<p>
The unique overhangs assign the position within the Level 1 assembly for the subsequent cloning steps. For some overhangs, additional nucleotides are appended. For example, overhangs in the 3’UTR position ($${\color{blue}{\text{GCTT}}\color{green}{\text{AA}}},{\color{blue}{\text{GGTA}}\color{green}{\text{A}}}$$) incorporate stop codons, while those at the tag positions ($${\color{blue}{\text{CCAT}}\color{green}{\text{G}},\color{green}{\text{GG}}\color{blue}{\text{AATG}},\color{blue}{\text{GCTT}}\color{green}{\text{TA}}}$$) include extra bases to ensure that fusion tags remain in frame.
</p>

<p>
For the domestication of new coding sequences, the original start and stop codons are removed. The start codon is reintroduced as part of the Golden Gate overhang for the CDS, which prevents double start codons, while the stop codon is incorporated into the 3’UTR/terminator sequence. This design allows for the modular addition of C-terminal tags.
</p>

<p>
For most Level 0 assemblies, the BsmBI enzyme is used. However, for new Connector/Integration site homology parts, the enzyme PaqCI is required along with the alternative Universal acceptor vector (pME_G_E_007_UAV_sfGFP_PaqCI). This is because these parts contain BsmBI recognition sites, which are necessary for the subsequent assembly of multi-gene constructs.
</p>
<br>

<html>
<p style="color:red;">red</p>
</html>

<html>
<table>
  <caption>
    <b>Table 1</b> Overhangs for creating new Level 0 parts via DNA synthesis/PCR<br>
    Level 1 overhangs are  indicated in blue. BsmBI sites ($${\color{brown}{\text{CGTCTC}}}/{\color{brown}{\text{GAGACG}}}$$) 
are indicated. For connector overhangs PaqCI sites are used.
  </caption>
  <tr>
    <td></td>
    <th scope="col" class="heman">He-Man</th>
    <th scope="col" class="skeletor">Skeletor</th>
  </tr>
  <tr>
    <th scope="row">Role</th>
    <td>Hero</td>
    <td>Villain</td>
  </tr>
  <tr>
    <th scope="row">Weapon</th>
    <td>Power Sword</td>
    <td>Havoc Staff</td>
  </tr>
  <tr>
    <th scope="row">Dark secret</th>
    <td>Expert florist</td>
    <td>Cries at romcoms</td>
  </tr>
</table>
</html>
<br>